examples/cpu/serving/triton/README.md

```diff
@@ -0,0 +1,69 @@
+# Serving BERT models with Triton Server and Intel® Extension for PyTorch optimizations
+
+## Description
+This sample provide code to integrate Intel® Extension for PyTorch (IPEX) with Triton Inference Server framework. This project provides custom python backend for IPEX and additional dynamic batching algorithm to improve the performance. This code can be used as performance benchmark for Bert-Base and Bert-Large models.
```

Comment:
Intel® Extension for PyTorch (IPEX) -> Intel® Extension for PyTorch*
Please avoid using the abbreviation `IPEX` in doc, always use `Intel® Extension for PyTorch*`.

Reply:
Removed IPEX abbreviation in README.md.